apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/transaction/Messaging.java

```diff
@@ -51,6 +53,7 @@
     private static final Logger log = getLogger(Messaging.class);
 
     private static PhasingPollService phasingPollService = CDI.current().select(PhasingPollService.class).get();
+    private static final AliasService ALIAS_SERVICE = CDI.current().select(AliasService.class).get();
```

Comment:
It's strongly recommended to use lazy lookup for CDI beans while TransactionType and Attachment class hierarchies are not refactored

Reply:
done